setup.py

```diff
@@ -43,15 +43,15 @@
         "packaging",
         "cryptography<3.4,>=3.2",
         "python-digitalocean>=1.16.0",
-        "PyJWT==1.7.1",     # azure-cli-core has pinned the PyJWT version as 1.7.1
+        "PyJWT>=2.1.0",
         "adal>=1.2.4",
         "azure-cli-core>=2.12.0",
         "azure-mgmt-compute>=5.0.0",
         "azure-mgmt-resource>=10.2.0",
         "azure-mgmt-cosmosdb>=6.0.0",
         "msrestazure >= 0.6.4",
         "azure-mgmt-storage>=16.0.0",
-        "azure-mgmt-sql>=0.11.0",
+        "azure-mgmt-sql==1.0.0",
```

Comment:
Rather than `==` pin, is it possible to express this as a `<=` pin?

Reply:
Yep sounds good @achantavy, I just made the change and tested it to be sure.